vignettes/custom_appearance.Rmd

```diff
@@ -381,4 +382,61 @@ basic_table(
   split_rows_by("STRATA1", label_pos = "hidden") %>%
   analyze("AGE", mean, format = "xx.x") %>%
   build_table(DM)
-```
\ No newline at end of file
+```
+
+### Cell, Label, and Annotation Wrapping
+
+The width of an `rtable` object can be customized by setting custom widths for cell contents, row labels, and titles/footers.
```

Comment:
this is somewhat misleading.

VTableTree objects (rtable objects are no longer a thing) dont' have a width, per se.

Everything you talk about in this section has to do with how the table is _rendered_, please change the text throughout to reflect that.

Reply:
I have updated the wording in this vignette. Please let me know if the new wording makes sense.